README.md

```diff
@@ -15,11 +30,51 @@ Current status:
 
 See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
 
-## Development tools
+
+## Contribute to `witan.models.demography`
+
+### General practices
+
+If you wish to contribute to `witan.models.demography`, please follow read the following guidelines.
+
+#### Github guidelines
+* **Fork this repository** (or clone it if you have writing writes to the reposotory).
+* **Create a new branch**. Let's try and keep the following naming conventions for branches:
+  * `feature/<what-this-branch-solves>`, example: `feature/exception-logging`
+  * `doc/<what-this-branch-solves>`, example: `doc/contributors-practices`
+  * `fix/<what-this-branch-solves>`, example: `fix/run-ccm-from-cli`
```

Comment:
interesting that the fix branch name is the same as the feature branch name - that makes sense though!